app/scripts/directives/blockEditor.js

```diff
@@ -11,7 +11,7 @@ angular.module('confRegistrationWebApp')
         $scope.showClearBtn = true;
         $scope.isAdmin = true;
         $scope.ruleTypeConstants = ruleTypeConstants;
-        
```

Comment:
This seems like it will never get run as the `$scope.block.type === 'paragraphContent'` condition above would make sure the type is not `checkboxQuestion`.